Microsoft.Toolkit.Uwp.SampleApp/SamplePages/Menu/Menu.bind

```diff
@@ -13,6 +13,77 @@
       <commands:NewFileCommand x:Key="NewFile" />
       <commands:GenericCommand x:Key="GenericCommand" />
       <converters:StringFormatConverter x:Key="StringFormatConverter" />
+
+      <Style TargetType="controls:MenuItem" x:Key="CustomDisabledStateMenuItemStyle">
```

Comment:
Is this style really needed to demonstrate the functionality of the Menu?

Reply:
This style has a minor difference compared to the default `Disabled` state - mainly the `Background` color. Default style keeps backwards compatibility by displaying the disabled item with the `Button`s disabled background color and the one on the sample sets a transparent background.

Reply:
I guess my question is more towards, should we have this style in the sample page? It seems to take up about half the sample, and it doesn't really show the functionality of the Menu, but how to apply a style.

Reply:
ping @r2d2rigo

Reply:
Sorry, I didn't know this question was for me! What kind of change are we looking at - removing it or improving it (more distinct disabled state)?

Reply:
No worries, it was my fault, I should have been more clear. The style does not seem to related to the functionality of the control and I feel it is not necessary to showcase what the control can do. If that is not the case, please feel free to let me know. Otherwise, we should remove the style and keep the sample simple

Reply:
Ok, so here's what I think:

- The default `Disabled` style for `MenuItem` mimics how it worked before (show the underliying `Button` as disabled) as to not introduce any UI changes. The style in the sample page is added to show that you can customize the `Disabled` state as you like, and it will work as expected.
- But on a second thought, I think that if someone wanted to style the `Disabled` state they wouldn't need any guidance through this sample, due to already having knowledge on how control templating works and expecting it to just work (that's how I found this bug!).

In the end, I think we can safely remove the change to the sample app and let users know that styling the `Disabled` state works by themselves.

Reply:
Agree 💯 , I'm good with merging this after the style has been removed :)